lib/modules/manager/scalafmt/index.ts

```diff
@@ -0,0 +1,14 @@
+import type { Category } from '../../../constants';
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import * as semverVersioning from '../../versioning/semver';
+
+export { extractAllPackageFiles, extractPackageFile } from './extract';
+
+export const supportedDatasources = [GithubReleasesDatasource.id];
+
+export const defaultConfig = {
+  fileMatch: ['\\.scalafmt.conf$'],
```

Comment:
Which names can these  `.scalafmt.conf` file have.  E.g. could it be `test.scalafmt.conf`?

If no, then it should be changed to: 
```suggestion
  fileMatch: ['(^|/)\\.scalafmt.conf$'],
```